ogre2/src/media/materials/programs/skybox_vs.glsl

```diff
@@ -0,0 +1,22 @@
+#version 330
+
+in vec4 vertex;
+in vec3 normal;
+uniform mat4 worldViewProj;
+
+out gl_PerVertex
+{
+    vec4 gl_Position;
+};
+
+out block
+{
+    vec3 cameraDir;
```

Comment:
Nit: the indentation is not consistent, sometimes 2, sometimes 4 spaces.

Reply:
fixed. 0823885